README.md

```diff
@@ -68,3 +68,33 @@ Go to admin/ and sign in with the admin password you just set.
 The repository unit tests and Docker images are built by CircleCI at https://circleci.com/gh/ministryofjustice/laa-legal-adviser-api.
 
 Deployment can be triggered via https://ci.service.dsd.io/job/DEPLOY-laalaa.
+
+## Releasing
+
+### Releasing to non-production
+
+> Currently, `staging` is the only non-production environment.
+
+1. Wait for [the Docker build to complete on CircleCI](https://circleci.com/gh/ministryofjustice/laa-legal-adviser-api) for the feature branch.
+1. Copy the `feature_branch.<sha>` reference from the `build` job's "Push Docker image" step. Eg:
+    ```
+    Pushing tag for rev [1ad776954b2e] on {https://registry.service.dsd.io/v1/repositories/laalaa/tags/dependabot-pip-django-filter-2.0.0.7243223}}
+    ```
+1. [Deploy `feature_branch.<sha>`](https://ci.service.dsd.io/view/LaaLaa/job/DEPLOY-laalaa/build?delay=0sec).
+    * `ENVIRONMENT` is the target environment, select "staging".
+    * `DEPLOY_BRANCH` is the branch that needs to be released (`dependabot-pip-django-filter-2.0.0` in the above example).
+    * `VERSION` is either `latest` for the last successful build on that branch, or a specific 7-character prefix of the Git SHA (`7243223` in the above example).
```

Comment:
For this repository, these variables seem to be different:

- `version` seems to be the concatenated `branch.sha` format
- `deploy_branch` seems to be correlating to the deploy repo's branch name.

<img width="350" alt="screen shot 2018-08-30 at 15 36 13" src="https://user-images.githubusercontent.com/1526295/44858677-738e0c00-ac6a-11e8-8ea9-717aaba3236f.png">